packages/schema-type/src/typescript/arrayAbiTypes.ts

```diff
@@ -1,17 +1,42 @@
 import { StaticAbiType } from "./schemaAbiTypes";
+import { isStaticAbiType } from "./staticAbiTypes";
 
-const arrayAbiTypePattern = /\[\]$/;
+export type ArrayAbiType = `${StaticAbiType}[]`;
+export type FixedArrayAbiType = `${StaticAbiType}[${number}]`;
+
+const arrayPattern = /\[\]$/;
+const fixedArrayPattern = /\[\d+\]$/;
 
 export type ArrayAbiTypeToStaticAbiType<T extends string> = T extends `${infer StaticAbiType}[]`
```

Comment:
this was already here so might not be worth refactoring, but according to our new implicit typescript style guide this should be lowercase